modules/setup-bias-monitoring.adoc

```diff
@@ -0,0 +1,83 @@
+:_module-type: PROCEDURE
+
+[id="setup-bias-monitoring_{context}"]
+= Before you begin
+
```

Comment:
Add prerequisites
== Prerequisites

- OpenShift AI is installed on your OpenShift cluster.

- You have cluster administrator permissions for the cluster where OpenShift AI is installed.

- You have installed the OpenShift command line interface, as described in Installing the OpenShift CLI https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/cli_tools/openshift-cli-oc#cli-installing-cli_cli-developer-commands

Reply:
maybe move prerequisites before the "Before you begin" section?